exonum/src/blockchain/tests.rs

```diff
@@ -574,14 +576,12 @@ fn test_dispatcher_already_deployed() {
         .unwrap_err();
     assert_eq!(err, DispatcherError::ArtifactAlreadyDeployed.into());
     // Tests that we cannot register artifact twice.
-    let result = execute_transaction(
+    let err = execute_transaction(
         &mut blockchain,
         TestDeploy { value: 1 }.sign(TEST_SERVICE_ID, keypair.0, &keypair.1),
-    );
-    assert_eq!(
-        result.0,
-        Err(DispatcherError::ArtifactAlreadyDeployed.into())
-    );
+    )
+    .unwrap_err();
```

Comment:
Minor nit: If this test will fail due to transaction success, current output will be like "Called `unwrap_err` on `Ok` value", but if we will compare not unwrapped result with `Err(..)` one line below, then the output will be like "got `Ok(())`, expected `Err(..)`", which is more helpful.